server/utils/sendEmail.js

```diff
@@ -0,0 +1,26 @@
+// External util for sending emails!
+// Currently attached in auth.ts upon login
+// Will be moved to the appropriate functionality in the future
+
+const sgMail = require("@sendgrid/mail");
+
+const sendEmail = async () => {
```

Comment:
Would be more extendable to have the message be passed as an argument rather than declared within the function, easier to have it serve multiple purposes

Reply:
Absolutely! This is the bare minimum to make it work until we discuss what we want the email to send over, templates, etc